Dockerfile

```diff
@@ -48,6 +48,7 @@ RUN \
   pacman -r /root-out -Sy --noconfirm \
     archlinux-keyring \
     bash \
+    busybox \
```

Comment:
Same as Fedora, why are we using busybox over cron?

Reply:
> Same as Fedora, why are we using busybox over cron?

https://github.com/linuxserver/docker-baseimage-fedora/pull/26#discussion_r1380376307